### DISCUSSION TITLE
Zeek playbook alerts

### DISCUSSION THREAD

**Discussion by ow111** [Security-Onion-Solutions/securityonion#4023] [Unsupported Versions] [answered]
After going through the training courses and the documentation I have not been able to find any examples of playbook rules including zeek logs. Does anyone have a rule they could share with me - looking to create quite a few of these for production use 

https://docs.securityonion.net/en/2.3/playbook.html?highlight=playbook%20zeek

Also regarding Playbook - is there functionality to apply rules to certain ip ranges/subnets - Still trying to learn as much as I can to do with SO!

TVM

**Comment by defensivedepth** [comment, marked as answer]
Currently the only Sigma rules pulled in by default are for windows (https://docs.securityonion.net/en/2.3/playbook.html#adding-additional-rulesets).

Here is an example Zeek rule: (https://github.com/SigmaHQ/sigma/blob/master/rules/network/zeek/zeek_rdp_public_listener.yml)

Be aware that we don't yet have all the Zeek mappings  complete - so if you see something that isn't working, let us know so that we can get it fixed.

RE:  apply rules to certain ip ranges/subnets

You would use a filter within the Sigma for that ie:
```
  filter:
      src_ip|startswith:
        - '192.168.'
        - '10.'
```

**Reply by ow111** [reply]
Thanks for the response - Answered all my questions 👍
